src/util/constants.js

```diff
@@ -1,5 +1,9 @@
 export const VIEW_INTERNAL_LINK_TAG = 'internallink';
 
+export const VIEW_INTERNAL_KEYWORD_ID_ATTRIBUTE = 'keywordid';
```

Comment:
no, because `PROPERTY_KEYWORD_ID` is the value of the keyword, and `VIEW_INTERNAL_KEYWORD_ID_ATTRIBUTE` is the name of the html attribute, and `MODEL_INTERNAL_KEYWORD_ID_ATTRIBUTE` is the name of the ckeditor-"model" property (which can have camel case, and the view-attribute cannot). They are used for different purposes in different contexts